src/Products/Common/Entity/Web/PageDescriptionEntity.cs

```diff
@@ -31,7 +31,7 @@ public class PageDescriptionEntity
         /// Page content data.
         /// </summary>
         [JsonProperty]
-        private string data;
+        public string data;
```

Comment:
Codacy found an issue: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Viewer-for-.NET-MVC/pullRequest?prid=5958475)